### PR TITLE
ENTERPRISE-298

### PR DESCRIPTION
**Pull request by wootencl**
Completion of [ENTERPRISE-298](https://pokitdok.atlassian.net/browse/ENTERPRISE-298) ticket which entailed the updating of the activities API endpoint client libraries code examples.

### CODE REVIEW

File: source/includes/_activities.md [hunk 1]
```diff
@@ -60,6 +120,151 @@ curl -XPUT -i -H "Content-Type: application/json"
 -d '{"transition": "cancel"}' https://platform.pokitdok.com/api/v4/activities/5317f51527a27620f2ec7533
 ```
 
+```python
+url = '/activities/574749250640fd22d719e13f'
+client.put(url, data={'transition': 'cancel'})
+```
+
+```ruby
```
Comment (jeffhoek) [inline source/includes/_activities.md]: if ruby client is no longer available shouldn't we just remove mentioning it altogether here?

**Comment by baldwindc** [comment]
If we're changing the name of the client variable from `pd` to `client`, can we change it here also? 

https://platform.pokitdok.com/documentation/v4/?python#security-and-authorization

Will probably want to make sure that we do that for all clients (keep our variable names consistent).

File: source/includes/_activities.md [hunk 2]
```diff
@@ -21,7 +21,34 @@ client.activities();
...
+client.activities();
+```
+
+> Example response:
+
+```json
+[
+  {
+    "units_of_work": 1,
+    "_type": "PlatformActivityModel",
```
Comment (dixonwh) [inline source/includes/_activities.md]: I can run a local test to confirm but the "_type" field should not be included in the API response as our internal->external model mapping does not support this field.

Comment (baldwindc) [inline source/includes/_activities.md]: I was curious, this is output from MOCKPAYER activities endpoint. 

image

**Comment by jeffhoek** [comment]
👍

File: source/includes/_activities.md [hunk 3]
```diff
@@ -21,7 +21,34 @@ client.activities();
...
+      "title": "Initializing"
+    },
+    "trading_partner_id": "PokitDok",
+    "id": "5745bbdd0640fd3a8186d5d6",
+    "transition_path": [
+      "process",
+      "complete"
+    ]
+  }
+]
```
Comment (baldwindc) [inline source/includes/_activities.md]: JSON should be wrapped in `{` and `}`

**Pull request merged**